### PR TITLE
SnowBoltz & Anime-Sugoi

### PR DESCRIPTION
**Pull request by Yusaki-Akito**
Thank you for participantion!

### CODE REVIEW

File: websites/S/SnowBoltz/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "Yusaki",
+    "id": "666246771785334784"
+  },
+  "service": "SnowBoltz",
+  "description": {
+    "en": "SnowBoltz The best shop for buying R$ in Thailand Easy Fast Secure",
+    "th_TH": "SnowBoltz ร้านรับเติม ROBUX (R$, โรบัค) ในเกม ROBLOX ที่ใช้ง่าย บริการจัดส่งอัตโนมัติรวดเร็วทันใจ เรทสุดคุ้ม 4 - 6 เเบบกลุ่มเเละ ID - Pass เรท 5 - 8"
```
Comment (Bas950) [inline websites/S/SnowBoltz/dist/metadata.json:9]: its just th not th_TH

File: websites/S/SnowBoltz/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "url": "snowboltz.net",
+  "version": "1",
```
Comment (Bas950) [inline websites/S/SnowBoltz/dist/metadata.json:12]: version 1.0.0

File: websites/S/SnowBoltz/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "logo": "https://i.imgur.com/xMeXqcE.png",
+  "thumbnail": "https://i.imgur.com/yYdlVDA.png",
+  "color": "#00ccff",
+  "tags": [
+    "Shop",
+    "R$",
+    "Thai",
+    "Thailand"
```
Comment (Bas950) [inline websites/S/SnowBoltz/dist/metadata.json:20]: tags should be lowercase

File: websites/S/SnowBoltz/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
...LHdCQUF3QixDQUFDO1lBQ2hELFlBQVksQ0FBQyxhQUFhLEdBQUcsU0FBUyxDQUFDO1lBQ3ZDLFlBQVksQ0FBQyxLQUFLLEdBQUcsSUFBSSxDQUFDO1NBQzNCO2FBQU0sSUFBSSxNQUFNLENBQUMsUUFBUSxDQUFDLFVBQVUsQ0FBQyxFQUFFO1lBQ3RDLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1lBQzVDLFlBQVksQ0FBQyxLQUFLLEdBQUcsSUFBSSxDQUFDO1lBQzFCLFlBQVksQ0FBQyxPQUFPLEdBQUcsY0FBYyxDQUFDO1lBQ3RDLFlBQVksQ0FBQyxhQUFhLEdBQUcsU0FBUyxDQUFDO1lBQ3ZDLFlBQVksQ0FBQyxLQUFLLEdBQUcsSUFBSSxDQUFDO1NBQzNCO2FBQU0sSUFBSSxNQUFNLENBQUMsUUFBUSxDQUFDLFNBQVMsQ0FBQyxFQUFFO1lBQ3JDLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1lBQzVDLFlBQVksQ0FBQyxLQUFLLEdBQUcsSUFBSSxDQUFDO1lBQzFCLFlBQVksQ0FBQyxhQUFhLEdBQUcsU0FBUyxDQUFDO1lBQ3ZDLFlBQVksQ0FBQyxPQUFPLEdBQUcsZUFBZSxDQUFDO1NBQ3hDO2FBQU8sSUFBSSxNQUFNLENBQUMsUUFBUSxDQUFDLFNBQVMsQ0FBQyxFQUFFO1lBQ3RDLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1lBQzVDLFlBQVksQ0FBQyxLQUFLLEdBQUcsSUFBSSxDQUFDO1lBQzFCLFlBQVksQ0FBQyxPQUFPLEdBQUcsb0JBQW9CLENBQUM7WUFDNUMsWUFBWSxDQUFDLGFBQWEsR0FBRyxTQUFTLENBQUM7U0FDeEM7S0FDRjtJQUVELElBQUksWUFBWSxDQUFDLE9BQU8sSUFBSSxJQUFJLEVBQUU7UUFDaEMsUUFBUSxDQUFDLFlBQVksRUFBRSxDQUFDO1FBQ3hCLFFBQVEsQ0FBQyxXQUFXLEVBQUUsQ0FBQztLQUN4QjtTQUFNO1FBQ0wsUUFBUSxDQUFDLFdBQVcsQ0FBQyxZQUFZLENBQUMsQ0FBQztLQUNwQztBQUNELENBQUMsQ0FBQyxDQUFDIn0=
```
Comment (Bas950) [inline websites/S/SnowBoltz/dist/presence.js:134]: Do NOT upload the JS file.

File: websites/S/SnowBoltz/presence.ts [hunk 5]
```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({
+  clientId: "666246771785334784"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+var params = new URL(document.location.href).toString(); 
+var uid = document.querySelector(
+  "#navbar_global > ul.navbar-nav.align-items-lg-center.ml-lg-auto > li.nav-item.d-none.d-lg-block.ml-lg-4.dropdown > div > a:nth-child(3)"
+);
+var uid1 = uid.textContent
```
Comment (Bas950) [inline websites/S/SnowBoltz/presence.ts:10]: use let or const instead of var

File: websites/S/SnowBoltz/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    "th_TH": "SnowBoltz ร้านรับเติม ROBUX (R$, โรบัค) ในเกม ROBLOX ที่ใช้ง่าย บริการจัดส่งอัตโนมัติรวดเร็วทันใจ เรทสุดคุ้ม 4 - 6 เเบบกลุ่มเเละ ID - Pass เรท 5 - 8"
+  },
+  "url": "snowboltz.net",
+  "version": "1",
+  "logo": "https://i.imgur.com/xMeXqcE.png",
+  "thumbnail": "https://i.imgur.com/yYdlVDA.png",
+  "color": "#00ccff",
+  "tags": [
+    "Shop",
+    "R$",
```
Comment (outapzaza) [inline websites/S/SnowBoltz/dist/metadata.json]: Fixed

**Comment by outapzaza** [comment]
image
image
image
image
image
image

**Comment by outapzaza** [comment]
I have made new Presences it name is Anime-Sugoi Thank you!

**Comment by outapzaza** [comment]
Anime-Sugoi Proof

Main Page
image
image

Ep Seletion 
image
image

Catalog
image
image

**Comment by outapzaza** [comment]
Sorry for edit something cus I got some bugs and forget some page Thank you for participantion!

**Comment by Yusaki-Akito** [comment]
> @Alanexei Can another reviewer check the thumbnails?

https://i.imgur.com/3wTOxd0.jpg

**Comment by Alanexei** [comment]
Oh well thank you, I'm on mobile so it's difficult to check. You don't have to change it

**Pull request merged**